app/views/inthenewsnewsitems/show.html.erb

```diff
@@ -86,11 +86,13 @@
 <% if @record.tags.present? %>
   <h3>Tags (<%= @record.tags.count %>)</h3>
   <div class="item_data">
-    <%= render partial: "inthenews/simple_association_list", locals: {
-      item: @record,
-      assoc: "tags",
-      path: nil
-    } %>
+    <ul>
+      <% @record.tags.each do |tag| %>
+        <li>
+          <%= link_to tag.name, inthenews_news_items_search_path(f: ["keywords|#{tag.name}"]) %>
```

Comment:
Is this where you're adding the unordered list?

Reply:
@sgar7 pretty much!

I open the `<ul>` on line 89, then I'm saying "for all the tags related to this database record, add a `<li>` with a link inside of it"  So line 92 is adding a link, rails-style, which will come out to `<a href="the_search_path">Tag Name</a>`

Reply:
Got it!